docs/tutorials/index.md

```diff
@@ -23,71 +23,63 @@ title: "Quickstart"
   -->
 
 
-In this quickstart, we will download Druid and set it up on a single machine. The cluster will be ready to load data
-after completing this initial setup.
+This quickstart gets you started with Apache Druid and introduces you to some of its basic features. 
+Following these steps, you will install Druid and load sample 
+data using its native batch ingestion feature. 
 
-Before beginning the quickstart, it is helpful to read the [general Druid overview](../design/index.md) and the
-[ingestion overview](../ingestion/index.md), as the tutorials will refer to concepts discussed on those pages.
+Before starting, you may want to read the [general Druid overview](../design/index.md) and
+[ingestion overview](../ingestion/index.md), as the tutorials refer to concepts discussed on those pages.
 
-## Prerequisites
+## Requirements
 
-### Software
+You can follow these steps on a relatively small machine, such as a laptop with around 4 CPU and 16 GB of RAM. 
 
-You will need:
+Druid comes with several startup configuration profiles for a range of machine sizes. 
+The `micro-quickstart`configuration profile shown here is suitable for early evaluation scenarios. To explore 
+Druid's performance or scaling capabilities, you'll need a larger machine.
 
-* **Java 8 (8u92+) or later**
-* Linux, Mac OS X, or other Unix-like OS (Windows is not supported)
-
-> **Warning:** Druid only officially supports Java 8. Any Java version later than 8 is still experimental.
->
-> If needed, you can specify where to find Java using the environment variables `DRUID_JAVA_HOME` or `JAVA_HOME`. For more details run the verify-java script.
+The configuration profiles included with Druid range from the even smaller _Nano-Quickstart_ configuration (1 CPU, 4GB RAM) 
+to the _X-large_ configuration (64 CPU, 512GB RAM). For more information, see 
```

Comment:
"X-Large", instead of "X-large" perhaps, to be consistent with "Nano-Quickstart".  In single-server.md, it uses "nano-quickstart", "xlarge" at the beginning, and "Nano-Quickstart", "X-Large" at the end.  If "Nano-Quickstart" is used here, I think "X-Large" should go with it.